### DISCUSSION TITLE
github ?s

### DISCUSSION THREAD

**Discussion by QgitboltQ** [notroj/cadaver#28] [General]
@notroj 
thanks again for helping me w/ the pull request :)

I had another question for you and there is no way to PM someone on github so I will just start this discussion.

so, when the owner of a github program repo (cadaver) merges one or more pull requests, how long will it take for those changes to reflect for everyone who uses the program?

I know that this probably depends on how the user initially installs the program (dpkg, appimage, snap, etc.) so maybe let's just stay with the context of dpkg?
it may be obvious that these changes simply reflect whenever the end-user updates their system via `sudo apt update && sudo apt dist-upgrade`.

cool story - I initially found `cadaver` and its man page typo when I spun up a Kali VM and entered `dpkg --list` in order to explore some unfamiliar and/or interesting-sounding packages that come with the distro by default. just excited to see my small change pop-up in an update. also a nice feeling that every other user of the program gets the same benefit. yay4opensource

just trying to understand how this whole github program repo update cycle works for the end-user following an update to the program

**Comment by notroj** [comment]
I love to see the excitement! :grinning: 
If you are using packages from your Linux distribution, they will typically get updated when an upstream project does a release, and then that release gets packaged up. So it depends on both how fast upstream does releases (varies wildly) and your distro, it may take weeks, months or years.
